net/nimble/controller/src/ble_ll_ctrl.c

```diff
@@ -1473,6 +1473,9 @@ ble_ll_ctrl_rx_reject_ind(struct ble_ll_conn_sm *connsm, uint8_t *dptr,
         ble_ll_ctrl_proc_stop(connsm, BLE_LL_CTRL_PROC_PHY_UPDATE);
         break;
 #endif
+    case BLE_LL_CTRL_PROC_DATA_LEN_UPD:
+        ble_ll_ctrl_proc_stop(connsm, BLE_LL_CTRL_PROC_DATA_LEN_UPD);
```

Comment:
We should have a comment here that in fact this is workaround for broken peer device - as per spec (Vol 6 Part B section 5.1.9) collisions in Data Length Update procedure are harmless and both devices should respond as normal.

Reply:
agree